src/pytorch_lightning/strategies/fully_sharded_native.py

```diff
@@ -224,6 +225,8 @@ def model_to_device(self) -> None:
 
     @contextlib.contextmanager
     def model_sharded_context(self) -> Generator:
+        # TODO: users should be able to initialize without wrapping their layers
+        # as shown in the example here: https://fairscale.readthedocs.io/en/stable/api/nn/fsdp.html
```

Comment:
Why add this TODO here? If relevant, create an issue for it.

Also this references fairscale instead of PyTorch

Reply:
removing this. Will create an issue, I added this to my personal backlog to investigate this first before creating an issue with POC.